src/main/resources/templates/messages.html

```diff
@@ -0,0 +1,61 @@
+{# The page to display user's messages #}
+{% extends "base/li-sidebar-page" %}
+{% block title %} Exception Null - Messages {% endblock %}
+
+{% macro msgSummary(person, lastMessaged, body, lastActive) %}
+<div class="message-summary">
+    <div class="title left">
+        {{ person }}
+    </div><div class="right">
+        {{ lastMessaged }}
+    </div>
+
+    <div class="truncate">
+        {{ body }}<br/>{{ lastActive }}
+    </div>
+</div>
+<hr />
+{% endmacro %}
+
+{% macro message(person, body, current) %}
+<div class={{ current ? "right-message" : "left-message" }}>
+    <div class="title">
+        {{ person }}:
+    </div>
+    <div class="message-body">
+        {{ body }}
+    </div>
+</div>
+{% endmacro %}
+
+{% block content %}
+
+<div class="messages">
+    <div class="card">
+        <div class="left-section">
+            <div class="subheader">
+                <i class="fa fa-pencil-square-o"></i>
+            </div>
+            <hr />
+            {{ msgSummary("Kodlee Yin", "10:57 PM", "Dependency injection? I solved that one awhile back. Sec") }}
+            {{ msgSummary("David Kim", "9:58 PM", "Do you have some time to help me? I'm stuck") }}
+            {{ msgSummary("Jonathan Li", "7:46 PM", "You: Thanks man, I owe ya; let me know next time if I can help") }}
+            {{ msgSummary("Brittney Hoy", "3:12 AM", "Thanks so much!") }}
+            {{ msgSummary("Michael Colins", "Sunday", "You: Let me know if you need any more help!") }}
+            {{ msgSummary("Sherlock Holmes", "Apr 14", "It's easy. All you have to do is re-read the problem.") }}
+        </div><div class="right-section">
```

Comment:
To avoid the spacing issue, use font-size: 0 on the parent (in this case, .card). Then you can newline the div. Just remember to reset the font size on the children.